### ISSUE TITLE
Feature Request: Navigate through torrents list via typing torrent name on keyboard

### ISSUE THREAD

**Issue by JustAndreww** [openscopeproject/TrguiNG#201]
For example, if you will take a look at this picture:
![image](https://github.com/openscopeproject/TrguiNG/assets/4374611/817d73d5-bd39-486a-8726-da0894d61e87)
you can spot 3 torrents currently in the list.

So when pressing 'M' button on keyboard 'MyName1' torrent will be selected, pressing 'M' again within a short period of time will look for torrents that starts with 'MM' and selecting them (case insensitive). If no 'MM' torrents available in the list the second torrent with 'M' in it's name will be selected - 'MoreFiles'.

Pressing 'A' will select first torrent in the list starting with 'A' in it's name (case insensitve).

Basically the same behavior you can see in default Windows Explorer when browsing through file list in a directory.

Thanks!
